test/certify/test_certify.py

```diff
@@ -1524,14 +1532,25 @@ def test_asdf_library_version_fail(jwst_serverless_state, jwst_data, caplog):
         CertifyScript(argv)()
         out = caplog.text
 
-    expected_out = f"""Certifying '{jwst_data}/jwst_fgs_distortion_bad_asdf_version.asdf' (1/1) as 'ASDF' relative to context 'jwst_0591.pmap'
+    if asdf.__version__ < "3.0.0":
+        expected_out = f"""Certifying '{jwst_data}/jwst_fgs_distortion_bad_asdf_version.asdf' (1/1) as 'ASDF' relative to context 'jwst_0591.pmap'
 Setting 'META.EXPOSURE.TYPE [EXP_TYPE]' = None to value of 'META.EXPOSURE.P_EXPTYPE [P_EXPTYP]' = 'FGS_IMAGE|FGS_FOCUS|FGS_INTFLAT|FGS_SKYFLAT|'
 File written with dev version of asdf library: 2.0.0.dev1213
 Checking JWST datamodels.
 ########################################
 0 errors
 1 warnings"""
+    else:
+        expected_out = f""" Certifying '{jwst_data}/jwst_fgs_distortion_bad_asdf_version.asdf' (1/1) as 'ASDF' relative to context 'jwst_0591.pmap'
```

Comment:
This file is invalid and no longer (in asdf 3.0) loads as a custom object (and instead is returned as a simple tagged dictionary). The main issue is that the file:
```yaml
#ASDF 1.0.0
#ASDF_STANDARD 1.2.0
%YAML 1.1
%TAG ! tag:stsci.edu:asdf/
--- !core/asdf-1.0.0
```
claims to be asdf-standard 1.2.0 which only supports `core/asdf-1.1.0` not `core/asdf-1.0.0`:
https://asdf-standard.readthedocs.io/en/1.0.3/generated/asdf-format.org/core/core-1.2.0.html
The expected output was updated to capture this.